qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -91,10 +88,15 @@ def __init__(
         Args:
             qubit: the qubit under test.
             delays: delay times of the experiments.
-            unit: Optional, time unit of `delays`.
+            unit: Optional, time unit of `delays`. \
             Supported units: 's', 'ms', 'us', 'ns', 'ps', 'dt'. The unit is \
             used for both T2Ramsey and for the frequency.
-            osc_freq: optional, oscillation frequency offset in Hz.
+            osc_freq: the oscillation frequency induced by the user. \
+            The frequency is given in Hz and scaled according to the \
+            reciprocal of the unit, \
+            e.g.,if unit='ms' then the frequency unit is KHz.
```

Comment:
A space is missing after "e.g.,"

Reply:
I don't understand the last sentence. It sounds like some scaling is done behind the scenes (correctly or not - I saw that there were some discussions about it, but I haven't followed them). It looks like an implementation detail that doesn't have to be exposed.

Reply:
Removed this sentence. I am planning to make a change in the units of frequency, but that will be in a separate PR.